packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/Abstractions/IExpressionApi.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Generator.CSharp.Expressions;
+
+namespace Microsoft.Generator.CSharp.ClientModel.Providers
+{
+    public interface IExpressionApi<T>
```

Comment:
here we need a constraint for `T`, at least it should be `where T : ValueExpression`.
Or it could be `where T : ScopedApi`

Reply:
added